hibernate-core/src/main/java/org/hibernate/engine/spi/ActionQueue.java

```diff
@@ -105,7 +105,7 @@
 	 */
 	private static final LinkedHashMap<Class<? extends Executable>,ListProvider> EXECUTABLE_LISTS_MAP;
 	static {
-		EXECUTABLE_LISTS_MAP = new LinkedHashMap<Class<? extends Executable>,ListProvider>( 8 );
```

Comment:
The above is a common misuse of `HashMap` or `LinkedHashMap`'s constructor. It doesn't take into consideration of `load refactor`. It is likely the map will be rehashed as long as the buckets number used surpasses 8 * 0.75. Seems the default constructor should be enough here.